### PR TITLE
Add missing fields for UpdateAccountRequest.cs

### LINKED ISSUE #542: UpdateAccountRequest missing BankAccountUUID as a parameter
Currently, it doesn't seem possible to do an UpdateAccountRequest to attach a bank account to an account

### PR DESCRIPTION
**Pull request by AlexandrosMor**
**Description**

**Tested scenarios**

**Fixed issue**:  
fixes #542

### CODE REVIEW

File: Adyen/Model/MarketPay/UpdateAccountRequest.cs [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Dotnet API Library
+ *
+ * Copyright (c) 2021 Adyen N.V.
```
Comment (wboereboom) [inline Adyen/Model/MarketPay/UpdateAccountRequest.cs:18]: image

**Review by wboereboom — CHANGES_REQUESTED** [review]
1 small  nitpick. 
Also: Do we need to update tests?

File: Adyen/Model/MarketPay/UpdateAccountRequest.cs (hunk 1 above)
Comment (AlexandrosMor) [reply Adyen/Model/MarketPay/UpdateAccountRequest.cs:18]: 🤣

Comment (AlexandrosMor) [inline Adyen/Model/MarketPay/UpdateAccountRequest.cs:18]:
```suggestion
 * Copyright (c) 2022 Adyen N.V.
```

**Review by zaiddreakh — APPROVED** [review]

**Pull request merged**